lib/3scale/backend/server/falcon.rb

```diff
@@ -4,6 +4,8 @@ module Server
       class Falcon
         extend ThreeScale::Backend::Server::Utils
 
+        CONFIG = 'falcon.rb'
```

Comment:
Isn't this the default value? I think even if we don't set it it will be used anyway?

Reply:
Yes, it is. We can remove it, I just add it to make it explicit like the `puma` file does.

https://github.com/3scale/apisonator/blob/3ab689d79d7d682ae89544ac46ffec2079a37e5c/lib/3scale/backend/server/puma.rb#L7